test/test.js

```diff
@@ -102,11 +106,22 @@ describe('Disabling', function () {
 });
 
 describe('Reporting errors', function () {
-  beforeEach(function() {
```

Comment:
Instead of deleting this beforeEach. Can you put your new test into a new `describe`? For example `describe('Reporting errors with custom URL`

Reply:
I wasn't sure what pattern you guys usually conformed to. Guess I chose the wrong one! Ha, is this what you were looking for or should I separate the new on out into it's own entirely?